app/src/main/java/ch/nevis/exampleapp/coroutines/dagger/ApplicationModule.kt

```diff
@@ -291,41 +303,45 @@ class ApplicationModule {
     fun provideInBandAuthenticationUseCase(
         clientProvider: ClientProvider,
         stateRepository: OperationStateRepository<UserInteractionOperationState>,
-        errorHandlerChain: ErrorHandlerChain,
         pinUserVerifier: PinUserVerifier,
         fingerprintUserVerifier: FingerprintUserVerifier,
         biometricUserVerifier: BiometricUserVerifier,
-        settings: Settings
+        devicePasscodeUserVerifier: DevicePasscodeUserVerifier,
+        settings: Settings,
+        onError: Consumer<AuthenticationError>
```

Comment:
Why not invoking `provideOnAuthenticationError` as was done before and is still done for the success callback?